/controller:
  ros__parameters:
    collision_aware: True
    collision_walls: True
    collision:
        floor: [3.0, 3.0, 0.1, 0.0, 0.0, -0.78, 0.0, 0.0, -0.785398163]
        wall1: [0.1, 5.0, 3.0, -0.5, -0.5, 0.0, 0.0, 0.0, 0.785398163]
        wall2: [5.0, 0.1, 3.0, 1.25, -1.25, 0.0, 0.0, 0.0, 0.785398163]
        wall3: [0.1, 5.0, 3.0, 1.25, 1.25, 0.0, 0.0, 0.0, 0.785398163]
        wall4: [5.0, 0.1, 3.0, -1.25, 1.25, 0.0, 0.0, 0.0, 0.785398163]
        stand: [0.2, 0.2, 0.53, 0.0, 0.0, -0.27, 0.0, 0.0, -0.785398163]
        linear_axis: [1.0, 2.15, 0.25, -0.2, -0.2, -0.655, 0.0, 0.0, -0.785398163]
        conveyor_control: [0.35, 0.48, 0.68, 0.85, -0.85, -0.44, 0.0, 0.0, -0.785398163]
        UR10_control: [0.8, 0.8, 0.8, 0.0, -1.3, -0.4, 0.0, 0.0, -0.785398163]
        conveyor: [2.6, 1.54, 0.55, 0.95, 0.95, -0.45, 0.0, 0.0, -0.785398163]
        specimen: [0.3, 0.3, 0.1, 1.0, 1.0, 0.0, 0.0, 0.0, -0.785398163]
        camera: [0.18, 0.03, 0.05, 0.0, 0.0, 0.04, 0.0, 0.0, 0.785398163]
    rectangle:
        y_min: -0.5
        x_offset: 0.5
        z_min: 0.0
        y_max: 0.5
        z_max: 1.0
        steps: 4.0
        pitch_adjust: True
        pitch_min: -0.1963
        pitch_max: 0.1963
        pitch_step: 0.1963
    cylinder_section: 
      R: 0.3
      step_size: 0.05
      min_angle: 1.047197551
      max_angle: 2.094395102
      x_offset: 0.6
      y_offset: 0.0
      z_offset: 0.0
    wait_milliseconds: 2000
    use_rectangle_trajectory: False